src/Uno.UI/UI/Xaml/Internal/RootVisual.cs

```diff
@@ -98,5 +103,52 @@ protected override Size ArrangeOverride(Size finalSize)
 
 			return finalSize;
 		}
+
+
+#if HAS_UNO
+		// Uno specific: To ensure focus is properly lost when clicking "outside" app's content,
+		// we set focus here. In case UWP, focus is set to the root ScrollViewer instead,
+		// but Uno does not have it on all targets yet.
+		private bool _isLeftButtonPressed = false;
+
+		private void RootVisual_PointerPressed(object sender, Windows.UI.Xaml.Input.PointerRoutedEventArgs e)
+		{
+			if (e.Handled)
```

Comment:
This case is actually impossible here as you subscribed to pointers using the event :) (same thing for all other handlers)

Reply:
Oh I see, if the event was handled, it would actually never bubble here 👍!